lib/rex/proto/rfb/client.rb

```diff
@@ -61,7 +61,13 @@ def handshake
 
     @minver = $2.to_i
 
-    our_ver = "RFB %03d.%03d\n" % [MajorVersion, @minver]
+		# Forces client version 3 to be used. This adds support
```

Comment:
Somewhat minor, but these are tabs.  msftidy should have caught this and it isn't clear why.